Interface/Harp.MultiPwm/Harp.MultiPwm.csproj

```diff
@@ -1,23 +1,23 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Title>Harp.MultiPwm</Title>
+    <Title>Harp - MultiPwm</Title>
     <Authors>harp-tech</Authors>
     <Copyright>Copyright © harp-tech and Contributors 2023</Copyright>
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
     <GeneratePackageOnBuild Condition="'$(Configuration)'=='Release'">true</GeneratePackageOnBuild>
     <Description>Bonsai Library containing interfaces for data acquisition and control of Harp MultiPwm devices.</Description>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <PackageType>Dependency;BonsaiLibrary</PackageType>
     <PackageTags>Harp MultiPwm Bonsai Rx</PackageTags>
-	<PackageProjectUrl>https://harp-tech.org</PackageProjectUrl>
+    <PackageProjectUrl>https://harp-tech.org</PackageProjectUrl>
     <RepositoryUrl>https://github.com/harp-tech/device.multipwm.git</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
     <PackageIcon>icon.png</PackageIcon>
     <PackageLicenseFile>LICENSE</PackageLicenseFile>
     <PackageOutputPath>..\bin\$(Configuration)</PackageOutputPath>
     <TargetFrameworks>net462;netstandard2.0</TargetFrameworks>
-	<AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
Not sure how this ended up here. I may have copied pasted it from somewhere else, but on the off chance i am wrong, were we planning some custom operator that relied on being able to manipulate unsafe blocks?